2025-Code/src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -177,9 +176,24 @@ public void changeMaxMotion(double mv, double ma, double ae) {
         PersistMode.kPersistParameters);
   }
 
+  private double safeAngle(double targetAngle) {
+    double safeAngle = targetAngle;
+    if (targetAngle < 180) {
+      if (targetAngle < FRONT_ARM_LOWEST_SAFE_LIMIT && targetAngle > FRONT_ARM_HIGHEST_SAFE_LIMIT) {
+        safeAngle = FRONT_ARM_HIGHEST_SAFE_LIMIT;
+      }
+    } 
+    else if (targetAngle > 180) {
+      if (targetAngle > BACK_ARM_LOWEST_SAFE_LIMIT && targetAngle < BACK_ARM_HIGHEST_SAFE_LIMIT) {
+        safeAngle = BACK_ARM_HIGHEST_SAFE_LIMIT;
+      }
+    }
+    return safeAngle;
+  }
+
   @Override
   public void periodic() {
-    updateArmAngle();
+    adjustArm(safeAngle(m_targetAngle));
```

Comment:
might be good to send this safe angle to network tables, but this is good for now